nf_core/pipeline-template/.github/workflows/download_pipeline.yml

```diff
@@ -42,13 +47,13 @@ jobs:
         run: |
           echo "REPO_LOWERCASE=${GITHUB_REPOSITORY,,}" >> ${GITHUB_ENV}
           echo "REPOTITLE_LOWERCASE=$(basename ${GITHUB_REPOSITORY,,})" >> ${GITHUB_ENV}
-          echo "REPO_BRANCH=${GITHUB_REF#refs/heads/}" >> ${GITHUB_ENV}
+          echo "REPO_BRANCH=${{ github.event.inputs.testbranch || 'dev' }}" >> ${GITHUB_ENV}
 
       - name: Download the pipeline
         env:
           NXF_SINGULARITY_CACHEDIR: ./
         run: |
-          nf-core download {% raw %} ${{ env.REPO_LOWERCASE }} \
```

Comment:
I think you need the raw exception here.

Reply:
That is exactly what we should talk about. I was not aware that you added them and believed that they are hallucinations from Copilot, because I had no other explanation how they ended up in the code 🫠. 

However, they interfere with the commands themselves, so we can't keep them around either. (At least not like so)

Reply:
how do they interfere with them? the workflow files looks okay for me otherwise:
https://github.com/nf-core/taxprofiler/actions/runs/7742219884/workflow? The only problem was that they added a trailing whitespace, but that is fixed now.

Reply:
Yes they did interfere, but that was my fault, because I just copied the file across from my local _tools_ repo to my local _testpipeline_ folder instead of using tools, so the Jinja escape statements were not removed. 

Github Copilot is also exonerated.